int16 LSpeed
int16 RSpeed
int16 LAddEN
int16 RAddEN
uint16 Voltage
uint16 State

